libevm/VMFactory.h

```diff
@@ -23,10 +23,11 @@ namespace dev
 namespace eth
 {
 
-enum class VMKind: bool
+enum class VMKind
 {
 	Interpreter,
-	JIT
+	JIT,
+	Smart,
```

Comment:
If you have comma in the end and want to add another element later you will not have to modify existing line. Diff looks better. But if you like it more without a comma :+1: